corehq/util/celery_utils.py

```diff
@@ -186,3 +190,21 @@ def get_running_workers(timeout=10):
         worker_names.extend(worker_info.keys())
 
     return worker_names
+
+
+class LoadBasedAutoscaler(Autoscaler):
+    def _maybe_scale(self, req=None):
```

Comment:
This function in the base class is https://github.com/celery/celery/blob/v3.1.18/celery/worker/autoscale.py#L84-L92